src/core/server/index.ts

```diff
@@ -229,6 +229,9 @@ export {
   SavedObjectsType,
   SavedObjectMigrationMap,
   SavedObjectMigrationFn,
+  getSortedObjectsForExport,
```

Comment:
optional: 
`getSortedObjectsForExport` doesn't return saved objects, but a stream of them.
`importSavedObjects` doesn't accept saved objects, but a stream of them.
what if we reflect this fact in the naming?
`exportToStream`/`importFromStream` for example